src/Covid19Dashboard/index.jsx

```diff
@@ -215,7 +209,7 @@ class Covid19Dashboard extends React.Component {
                 />
               </div>
               <div className='covid19-dashboard_visualizations'>
-                <WorldMapChart {...this.props} />
+                <WorldMapChart rawMapData={this.props.rawData.filter(location => (location.project_id === 'open-JHU' && (location.confirmed.length || location.deaths.length)))} />
```

Comment:
what do you need `&& (location.confirmed.length || location.deaths.length)` for? 😄

Reply:
nvm, i see why. probably should also be checked for the illinois map, but whatever

Reply:
this logic https://github.com/uc-cdis/data-portal/pull/685/files#diff-891735d0760d4f7e2be89d9467b3ba0fL266 was filtering out no data locations. I'm just moving this into here

Reply:
> nvm, i see why. probably should also be checked for the illinois map, but whatever

i'll leave this task to you :smirk: